app/components/display/alert_component.rb

```diff
@@ -1,9 +1,10 @@
 class Display::AlertComponent < ViewComponent::Base
-    def initialize(message: nil, closable: true, type: "info", auto_close_delay: nil)
+    def initialize(message: nil, closable: true, type: "info", auto_close_delay: nil, style: nil)
```

Comment:
you will need to explain to me why this change, and add an argument `style` to just have` justify-content: space-between`. Is there no better way to do this?

Reply:
I tried with other methods, but it didn't work, so that's why I used this method to add a style to the component